### PR TITLE
Consolidate git setup documentation

### LINKED ISSUE #9091: Add "The Kubernetes Github Workflow" to docs/devel/
I'd like to suggest converting the content of "[The Kubernetes Github Workflow](https://docs.google.com/presentation/d/1HVxKSnvlc2WJJq8b9KCYtact5ZRrzDzkWgKEfm0QO_o/pub?start=false&loop=false&delayms=3000&slide=id.g8c9142d48_00)" from google docs preso to markdown for inclusion in `docs/devel/`.

These slides contain important info for new contributors which would ideally appear right in the dev docs and also be under version control.

Happy to submit a PR for this (with a little guidance on what file it should appear in)!

### PR DESCRIPTION
**Pull request by JeffPaine**
Closes #9091

### CODE REVIEW

File: docs/devel/development.md [hunk 1]
```diff
@@ -8,23 +8,62 @@ Official releases are built in Docker containers.  Details are [here](../../buil
...
-We highly recommend to put kubernetes' code into your GOPATH. For example, the following commands will download kubernetes' code under the current user's GOPATH (Assuming there's only one directory in GOPATH.):
+Below, we outline one of the more common git workflows that core developers use. Other git workflows are also valid.
+
+### Visual overview
+![Git workflow](git_workflow.png)
+
+### Fork
+
+1. Go to https://github.com/GoogleCloudPlatform/kubernetes
+2. Click the "Fork" button (in top right)
```
Comment (quinton-hoole) [inline docs/devel/development.md]: nit: In top right corner?  At the top right?

Comment (JeffPaine) [inline docs/devel/development.md]: Fixed, PTAL :)

**Comment by JeffPaine** [comment]
Updated, PTAL, thanks!

**Comment by quinton-hoole** [comment]
LGTM.  Apologies for the delay.

**Comment by JeffPaine** [comment]
Thanks and no worries! Happy to help.

**Pull request merged**